<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessesChecker</class>
 <widget class="QWidget" name="ProcessesChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ProcessesCheckLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Client IP:&gt; </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProcessesCheckEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ProcessesCheckBrowser">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>81</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
